src/main/scala/encry/modifiers/NodeViewModifier.scala

```diff
@@ -13,7 +13,6 @@ sealed trait NodeViewModifier extends BytesSerializable {
 
   val modifierTypeId: ModifierTypeId
 
-  //todo: check statically or dynamically output size
   def id: ModifierId
 
   def encodedId: String = Base58.encode(id)
```

Comment:
You can use object Algos from "encry.settings.Algos" instead of Base58 from "scorex.crypto.encode.Base58"

Reply:
Don't wonna touch logic in this pr.